src/test/resources/application.yml

```diff
@@ -30,12 +30,14 @@ store:
         blocks-partition-size: 10
         use-virtual-thread-for-batch-processing: true
         use-virtual-thread-for-event-processing: true
+    core:
+        enabled: ${INDEXER_ENABLED:true}
 
 ################## Aggregation Configuration ###########
 aggregation:
     modules:
         addressTxCount-enabled: ${AGGREGATOR_ADDRESS_TX_COUNT_ENABLED:true}
-        poolstatus-enabled: ${POOL_STATUS_ENABLED:true}
+        poolaggregation-enabled: ${POOL_AGGREGATION_ENABLED:true}
         uniqueAccountCount-enabled: ${AGGREGATOR_UNIQUE_ACCOUNT_ENABLED:true}
```

Comment:
can I suggest avoiding env var here?

Spiring accepts and ovverride props based on property name already https://docs.spring.io/spring-boot/docs/1.5.6.RELEASE/reference/html/boot-features-external-config.html#boot-features-external-config-relaxed-binding

This helps on the devops/infra size. Just stick the default there.

Reply:
Sure, I'll remove it